examples/absorbed_SynIC.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+import numpy as np
+import astropy.units as u
+from astropy.io import ascii
+
+import naima
+
+## Model definition
+
+from naima.models import InverseCompton, Synchrotron, ExponentialCutoffPowerLaw, BrokenPowerLaw, EblAbsorptionModel
+
+# Define the redshift of the source, and absorption model
+redshift = 0.06 * u.dimensionless_unscaled
+EBL_transmitance = EblAbsorptionModel(redshift, 'Dominguez')
+
+def ElectronSynIC(pars, data):
+
+    # Match parameters to ECPL properties, and give them the appropriate units
+    amplitude = 10**pars[0] / u.eV
+    e_break = (10**pars[1]) * u.TeV
+    alpha1 = pars[2]
+    alpha2 = pars[3]
+    B = pars[4] * u.uG
+
+    # Initialize instances of the particle distribution and radiative models
+    BPL = BrokenPowerLaw(amplitude, 1. * u.TeV, e_break, alpha1, alpha2)
+    
+    # Compute IC on CMB and on a FIR component
+    IC = InverseCompton(
+        BPL,
+        seed_photon_fields=['CMB'],
+        Eemin=10 * u.GeV)
+    SYN = Synchrotron(BPL, B=B)
+
+    # compute flux at the energies given in data['energy']
+    model = (EBL_transmitance.transmission(data) * IC.flux(data, distance=1.0 * u.kpc) + SYN.flux(data,
+                                                      distance=1.0 * u.kpc))
```

Comment:
Can you break this line right after the `+` and align that with the start of the parens?